src/lib/components/event.svelte

```diff
@@ -27,20 +27,23 @@
   class:pending
   class:active={$page.path.includes(href)}
 >
-  <article class="p-4 w-full">
-    <h2 class="mb-2 {tag}">
-      <EventLabel color={classification}>
-        {name}
-      </EventLabel>
-    </h2>
-    <p class="text-sm">
-      <Icon icon={faCalendar} class="inline" />
-      {formatDate(timeStamp)}
-    </p>
+  <article class="flex gap-4 items-center p-4">
+    <p class="w-5 text-center text-gray-500">{id}</p>
+    <div class="w-full">
```

Comment:
This could be `grow` and if it's not a high enough "growth" you could use `grow-[x]`